lightning/src/ln/channel.rs

```diff
@@ -766,7 +768,8 @@ impl<Signer: Sign> Channel<Signer> {
 				holder_selected_contest_delay: config.own_channel_config.our_to_self_delay,
 				is_outbound_from_holder: true,
 				counterparty_parameters: None,
-				funding_outpoint: None
+				funding_outpoint: None,
+				opt_anchors: None,
```

Comment:
(can't comment on the relevant line), but can you add a check in the Channel deserialization logic to return `Err(InvalidValue)` if `opt_anchors` is set and add a comment that we should relax the check based on `channel_type` when `ChannelTypeFeatures` supports anchors.

Reply:
done